Buildings/Fluid/Chillers/ModularReversible/BaseClasses/RefrigerantCycleHeatRecovery.mo

```diff
@@ -0,0 +1,159 @@
+within Buildings.Fluid.Chillers.ModularReversible.BaseClasses;
+model RefrigerantCycleHeatRecovery
+  "Refrigerant cycle model for cooling-only or heat recovery chiller models"
+  extends Buildings.Fluid.HeatPumps.ModularReversible.BaseClasses.PartialModularRefrigerantCycle(
+    use_rev=false);
+  parameter Boolean have_switchover=false
```

Comment:
may change to `have_switchOver`?

Reply:
The naming was chosen for consistency with the same parameter in `Buildings.Fluid.Chillers.BaseClasses.PartialElectric`.